wtr-utils.js

```diff
@@ -9,6 +9,14 @@ const group = process.argv.indexOf('--group') !== -1;
 
 const NO_UNIT_TESTS = ['vaadin-icons', 'vaadin-lumo-styles', 'vaadin-material-styles'];
 
+/**
+ * Check if lockfile has changed.
+ */
+const isLockfileChanged = () => {
+  const log = execSync('git diff --name-only origin/master HEAD').toString();
```

Comment:
Looks like we don't run a pipeline on master itself after merges? If so then this checks out.

Though we might want to consider doing the above, I have seen side effects breaking master when branches were not rebased before merging. In Gitlab there also was a rule to disallow merging if the PR was not rebased.